src/App.tsx

```diff
@@ -15,49 +17,121 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  NONE,
+  ALPHABET,
+  LENGTH,
+}
+
+type ReorderOptions = {
+  sortType: SortType;
+  isReversed: boolean;
+};
+
+export function getReorderedGoods(
+  goods: string[],
+  { sortType, isReversed }: ReorderOptions,
+) {
+  const visibleGoods = [...goods];
+
+  if (sortType !== SortType.NONE) {
+    visibleGoods.sort((good1, good2) => {
```

Comment:
🐞 Instead of using a magic number like `!== SortType.NONE`, consider using a more descriptive enumeration value, such as `SortType.UNSORTED`. This will improve the code readability.